### PR TITLE
PRIME-1847 GIS Updated Phone/Email in MoH Keycloak

### PR DESCRIPTION
**Pull request by mtpultz**
* Update Phone/Email in MoH Keycloak
* Remove organization and role
* Update user messaging on GIS role missing when authenticating with LDAP

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html [hunk 1]
```diff
@@ -40,23 +40,34 @@
...
           <app-alert type="danger"
                      icon="error_outline">
             <ng-container #alertContent
                           class="alert-content">
-              <ng-container *ngIf="locked; else oneMoreAttempt">
-                Your account is now locked due to too many failed login attempts. If you think this is a mistake please
-                contact Reg Admin at <a href="tel:12509529137">(250) 952-9137</a> or
-                <a href="mailto:HLTH.REGISTRIESADMIN@gov.bc.ca">HLTH.REGISTRIESADMIN@gov.bc.ca</a>.
+              <ng-container *ngIf="unauthorized; else authenticationFailed">
+                You have been authenticated, but do not sufficient permission to proceed. If you think this is a
```
Comment (Paahn) [inline prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html:49]: add "have" ie 
`You have been authenticated, but do not have sufficient permission to proceed. If you think this is a`

**Review by Paahn — CHANGES_REQUESTED** [review]

**Review by Paahn — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/ViewModels/Parties/GisLdapUserViewModel.cs [hunk 2]
```diff
@@ -4,6 +4,6 @@ public class GisLdapUserViewModel
     {
         public string Unlocked { get; set; }
         public string GisUserRole { get; set; }
-        public bool Success { get => GisUserRole == "GISUSER"; }
+        public bool Success { get => GisUserRole == PrimeEnvironment.MohKeycloak.GisUserRole; }
```
Comment (james-hollinger) [inline prime-dotnet-webapi/ViewModels/Parties/GisLdapUserViewModel.cs:7]: 👍

File: prime-dotnet-webapi/Controllers/GisController.cs [hunk 3]
```diff
@@ -175,6 +175,10 @@ public async Task<ActionResult> LdapLogin(int gisId, LdapLoginPayload payload)
             {
                 return Ok();
             }
+            if (ldapResponse.Authenticated == "true")
+            {
+                return Forbid();
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/GisController.cs:180]: perhaps a quick comment here saying that the user is authenticated but does not have the role?

**Review by james-hollinger — APPROVED** [review]
L GIS TM

File: prime-dotnet-webapi/Controllers/GisController.cs (hunk 3 above)
Comment (mtpultz) [reply prime-dotnet-webapi/Controllers/GisController.cs:181]: Thanks @james-hollinger updated

**Pull request merged**